cardano-cli/src/Cardano/CLI/EraBased/Genesis/Option.hs

```diff
@@ -490,7 +490,7 @@ pSecurityParam =
   Opt.option nonZeroReader $
     mconcat
       [ Opt.long "security-param"
-      , Opt.metavar "NonZero WORD64"
+      , Opt.metavar "Non zero WORD64"
```

Comment:
Most of metavariables in help here are using screaming snake case, so let's make it that way.
```suggestion
      , Opt.metavar "NON_ZERO_WORD64"
```